modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -92,7 +92,7 @@ public void setUp() throws Exception {
 		_objectDefinition1 = ObjectDefinitionTestUtil.publishObjectDefinition(
 			Collections.singletonList(
 				ObjectFieldUtil.createObjectField(
-					, "String", true, true, null,
+					"Text", "String", true, true, null,
```

Comment:
I think this was performed in the previous commits. Maybe we should remove this part from both commits to avoid noise